kubernetes/annotations_and_labels.md

```diff
@@ -1,21 +1,33 @@
 # Kubernetes Annotations and Labels
 
-## Common Labels
+## Common Labels (for host cluster services)
 
 - `app` - value should contain the name of the application. Should be applied
-  to every Kubernetes resource associated with an application. This should be
-  also used as replicas selector. E.g. `app=kvm-operator`.
+  to every Kubernetes resource associated with an application. This should
+  also be used as replicas selector. E.g. `app=kvm-operator`.
 - `giantswarm.io/cluster` - value should contain guest cluster ID which this
   object is part of. E.g. `giantswarm.io/cluster=eggs2`.
-  operator managing the object. E.g. `giantswarm.io/managed-by=kvm-operator`.
 - `giantswarm.io/managed-by` - value should contain repository name of the
+  operator managing the object. E.g. `giantswarm.io/managed-by=kvm-operator`.
 - `giantswarm.io/organization` - value should contain guest cluster's
-organization ID as displayed in the front-end
-`giantswarm.io/organization=track-hunter`.
+  organization ID as displayed in the front-end
+  `giantswarm.io/organization=track-hunter`.
 - `giantswarm.io/randomkey` - value should contain type of the randomkey stored
   in secret. E.g. `giantswarm.io/randomkey=encryption` for guest API -> etcd
   encryption at rest.
 
+## Common Labels (for guest cluster services)
+
+- `app` - value should contain the name of the application. Should be applied
+  to every Kubernetes resource associated with an application. This should 
+  also be used as replicas selector. E.g. `app=nginx-ingress-controller`.
+  Exceptions can be made to accomodate for adherence to existing selectors
+  upstream.
```

Comment:
Why not just extending the description in previous section?

Reply:
because I do not want to mix host cluster and guest cluster things, they are different, one the customer sees and the other not

Reply:
I don't see the argument here, why not add all labels on hostcluster aswell? 
It would increase visibility of which things we actively manage and which are `system` on hostcluster IMO.

Reply:
I do not argument against adding labels to host cluster, I just argue that host cluster services have a different need and use for labels than guest cluster ones. For host cluster we do have the set above already, but it seems it's not used that much.

We can however talk about if it's enough to have names of operators there or if we need additional labels.

It's just a bit off topic for this PR as I mainly care about solving a customer facing (and support) issue.